cmd/cephcsi.go

```diff
@@ -90,6 +90,7 @@ func init() {
 		"skip image flattening if kernel support mapping of rbd images which has the deep-flatten feature")
 
 	flag.BoolVar(&conf.Version, "version", false, "Print cephcsi version information")
+	flag.BoolVar(&conf.EnableProfiling, "enableprofiling", true, "enable go profiling")
```

Comment:
I am not sure we want it enabled by default. I think it is fine, but you probably should add a commit that adds the option for the `deploy/` and `charts/` yaml files.

Reply:
I dont think enabling `Profiling` by default is a good idea and also invite some security issues. Especially when we expose `/debug` in a cluster which can be used by any one in the cluster or outside depends on the production cluster setup.

Reply:
yes, I am working on that to add enable options to the yaml files.

Reply:
disable it by default in code. if required enable it in yaml files

Reply:
add a check to start the HTTP server if the `enablegrpcmetrics` is disabled and profiling is enabled

Reply:
This has been done already.